sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -246,18 +245,14 @@ export interface RulesResponse extends Array<RuleDescription>, Response {
 
 // @public
 export interface Sender {
-    cancelScheduledMessage(sequenceNumber: Long, options?: OperationOptions): Promise<void>;
-    cancelScheduledMessages(sequenceNumbers: Long[], options?: OperationOptions): Promise<void>;
+    cancelScheduledMessages(sequenceNumbers: Long | Long[], options?: OperationOptions): Promise<void>;
     close(): Promise<void>;
     createBatch(options?: CreateBatchOptions): Promise<ServiceBusMessageBatch>;
     entityPath: string;
     isClosed: boolean;
     open(options?: SenderOpenOptions): Promise<void>;
-    scheduleMessage(scheduledEnqueueTimeUtc: Date, message: ServiceBusMessage, options?: OperationOptions): Promise<Long>;
-    scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: ServiceBusMessage[], options?: OperationOptions): Promise<Long[]>;
-    send(message: ServiceBusMessage, options?: OperationOptions): Promise<void>;
-    send(messages: ServiceBusMessage[], options?: OperationOptions): Promise<void>;
-    send(messageBatch: ServiceBusMessageBatch, options?: OperationOptions): Promise<void>;
```

Comment:
> Replaced overloads for send() with input parameter supporting union type to be consistent with how we are dealing with other methods that take both single & multi items

Why not retain this send as 
```ts
    sendMessages(message: ServiceBusMessage, options?: OperationOptions): Promise<void>;
    sendMessages(messages: ServiceBusMessage[], options?: OperationOptions): Promise<void>;
    sendMessages(messageBatch: ServiceBusMessageBatch, options?: OperationOptions): Promise<void>;
```
and add the new ones as follows?
```ts
    receiveDeferredMessages(sequenceNumber: Long, options?: OperationOptions): Promise<ReceivedMessageT[]>;
    receiveDeferredMessages(sequenceNumbers: Long[], options?: OperationOptions): Promise<ReceivedMessageT[]>;
```

Reply:
This is up for debate and the final result should probably go to our guidelines :)

Personally, I like the single method model because it reduces clutter in the editor suggestions. I'd rather see a single `receiveDeferredMessages` in my suggestions and then when I add the `(` see that I can pass either a single value or an array, instead of seeing the two suggestions, making me pause and think as to which ones I should be using. 

From a API reference docs perspective, it is one less method to document if we don't follow the overloads pattern.

cc @xirzec, @bterlson, @chradek

Reply:
I also dislike overloads since we don't really have them. :)

Is it really that useful to support both singular and array call patterns? Arrays are cheap in JS, so my default is to say only take an array and someone can box a singular value if they need to.

Reply:
> Is it really that useful to support both singular and array call patterns?

I like removing singular :)

Reply:
In Event Hubs we just have a single `sendBatch` method that accepts either a batch or an array of events and don't have a single event overload. I'm also in favor of removing the singular call pattern if the suffix is "messages" since that is plural :)

Reply:
> so my default is to say only take an array and someone can box a singular value if they need to.

What benefit do we get by doing this vs having the signature reflect union type?

Reply:
@xirzec - from discussions in the team, sending single messages (ie, dispatching commands) is a common enough pattern in ServiceBus that we want to support it without any extra work, small as it might be.

This is in contract to EventHubs where we were (and are) pretty sure that sending large batches of messages to optimize throughput was more common.